Sources/REventLogger/REvent.swift

```diff
@@ -47,3 +43,23 @@ struct REvent: Codable {
         self.info = info
     }
 }
+
+/// Describe the type of the event
+enum EventType: String, Codable {
+    case critical, warning
+}
+
+extension REvent {
+    func getEventIdentifier() -> String {
```

Comment:
1. This could be a computed property `var eventId: String` 
2. `String(describing:` not required since `appVersion` is String type